exercises/robot-simulator/canonical-data.json

```diff
@@ -51,10 +54,12 @@
       ]
     },
     {
-      "description": "rotates the robot's direction 90 degrees clockwise",
+      "description": "Rotate clockwise",
+      "comments": [ "rotates the robot's direction 90 degrees clockwise" ],
       "cases": [
         {
-          "description": "changes the direction from north to east",
+          "description": "north to east",
```

Comment:
We already know it changes direction, since the parent is "Rotate clockwise".
For a succinct unit test identifier, I feel the [1.] below doesn't lose any information 
over @sshine suggestion [2.] and original [3.]
```
1. rotate_clockwise_north_to_east
2. rotate_90_degrees_clockwise_change_direction_from_north_to_east
3. rotates_the_robot's_direction_90_degrees_clockwise_changes_the_direction_from_north_to_east